src/transaction/log_recovery_redo_parallel.cpp

```diff
@@ -222,6 +222,13 @@ namespace cublog
     // which should help to only allocate/reserve once
     jobs_vec.reserve (PARALLEL_REDO_JOB_VECTOR_RESERVE_SIZE);
 
+#if defined(NDEBUG)
+    if (prm_get_bool_value (PRM_ID_RECOVERY_PARALLEL_TASK_DEBUG))
+      {
+	er_log_debug (ARG_FILE_LINE, "[PARALLEL RECOVERY] Call to parallel execute with task index: %d. ", m_task_idx);
```

Comment:
```
- keep an on-stack counter about the number of jobs the task has executed, debug mode only
- use the parameter in redo_parallel::redo_task::execute to log the number of jobs executed along with m_task_idx, debug mode only - comment the motives behind this logging
```
This is needed to ensure that the task really does process log records. Entering the `execute` function does not guarantee that the task also processes log records.